src/jail_run.c

```diff
@@ -269,6 +269,7 @@ jail_run(
             "setresuid(%u,%u,%u) error: %s", (guint)creds->ruid,
             (guint)creds->euid, (guint)creds->suid, strerror(errno)));
     } else {
+        fflush(0);
```

Comment:
NULL?

Reply:
In c literal 0 is automatically promoted to null pointer. Caveat varargs where compiler does not know the intent.

Reply:
I know, but I'm not a compiler :) Zero is just a number without any meaning attached to it. NULL has additional semantics (hinting that it's a pointer).

Reply:
NULL it is